js/web/test/e2e/node-test-main.js

```diff
@@ -0,0 +1,8 @@
+const ort = require('onnxruntime-web');
+const testFunction = require('./common');
+
+it('Browser E2E testing - WebAssembly backend', async function () {
+  await testFunction(ort, { executionProviders: ['wasm'] });
```

Comment:
do we need to set ort.env.wasm.numThreads explicitly for a machine having only 1 or 2 logical CPUs?

Reply:
short answer is: no

a longer answer is: it's actually depend on the value `navigator.hardwareConcurrency` from browser. From [MDN document](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorConcurrentHardware/hardwareConcurrency), not all browsers support this property, for example, in safari this will be `undefined`, so that means it will fallback to single thread. if the value is a number, we set numThreads to `Math.ceil((numCpuLogicalCores || 1) / 2)` and its max value is 4.